components/crud-web-apps/common/backend/kubeflow/kubeflow/crud_backend/api/isvc.py

```diff
@@ -0,0 +1,8 @@
+from . import utils, events
+
+
+def list_inference_service_events(name, namespace):
```

Comment:
I'd propose to avoid introducing a handler for the InferenceServices in the common backend code.

This would mean we'd need to add more handlers for list/delete/get/logs as well, which as a result will make the MWA depend even more in this common code. I'd propose to instead keep on using the `custom_resource.py` file's handlers for the ISVC logic in the MWA

Reply:
ACK! I've just removed that part and will add the changes to the follow-up PR, which will introduce the `EVENTS` tab, as they'll relate to the MWA backend code.